src/Microsoft.OData.Core/ODataErrorDetail.cs

```diff
@@ -22,5 +24,16 @@ public sealed class ODataErrorDetail
         /// <summary>Gets or sets the target of the particular error.</summary>
         /// <returns>For example, the name of the property in error</returns>
         public string Target { get; set; }
+
+        /// <summary>
+        /// Serilization to Json format string.
+        /// </summary>
+        /// <returns>The string in Json format</returns>
+        public override string ToString()

```

Comment:
And the tostring is only used in Json and JsonLight cases.

Reply:
how do you know? it is a public method on a public class?

---
In reply to: [134844889](https://github.com/OData/odata.net/pull/951#discussion_r134844889) [](ancestors = 134844889)

Reply:
i thought this was going to be renamed as ToJson()?

---
In reply to: [135375128](https://github.com/OData/odata.net/pull/951#discussion_r135375128) [](ancestors = 135375128,135333869,134844889)